test/benchmarks/kubernetes_state/main.go

```diff
@@ -231,6 +230,10 @@ func main() {
 	pprof.StopCPUProfile()
 
 	cancel()
-
 	fmt.Printf("KSMCheck.Run() returned %v in %s\n", err, elapsed)
+
+	if err = file.Close(); err != nil {
+		log.Println("failed to close \"cpuprofile.pprof\": %v\n", err)
```

Comment:
As you’re using `%v` verb, you should use `Printf` instead:
```suggestion
		log.Printf("failed to close \"cpuprofile.pprof\": %v\n", err)
```